app/src/main/java/com/alexbar/layoutbasic/movies_app/nav/BottomNavigationGraph.kt

```diff
@@ -0,0 +1,72 @@
+package com.alexbar.layoutbasic.movies_app.nav
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.alexbar.layoutbasic.movies_app.model.Media
+import com.alexbar.layoutbasic.movies_app.screen.detail.MediaDetailScreen
+import com.alexbar.layoutbasic.movies_app.screen.favorites.FavoritesMediaScreen
+import com.alexbar.layoutbasic.movies_app.screen.mix.MixMediaScreen
+import com.alexbar.layoutbasic.movies_app.screen.trending.TrendingMediaScreen
+import com.alexbar.layoutbasic.movies_app.utils.getVoteAverageFormatted
+import com.alexbar.layoutbasic.ui.theme.AppBackground
+
+@Composable
+fun BottomNavigationGraph(navController: NavHostController) {
```

Comment:
I have made some changes regarding this class :

```
@Composable
fun BottomNavigationGraph(navController: NavHostController) {
    NavHost(
        navController = navController,
        startDestination = Screens.TrendingMediaScreen
    ) {
        composable(route = Screens.TrendingMediaScreen) {
            TrendingMediaScreen(
                favorites = emptyList(),
                onMediaItemSelected = { media ->
                    navigateToMediaDetail(navController, media)
                }
            )
        }
        composable(route = Screens.FavoritesMediaScreen) {
            FavoritesMediaScreen(favorites = emptyList()) { _ -> }
        }
        composable(route = Screens.MixMediaScreen) {
            MixMediaScreen()
        }
        composable(
            route = "${Screens.DetailMediaScreen}/{title}/{description}/{voteAverage}",
            arguments = listOf(
                navArgument("title") { },
                navArgument("description") { },
                navArgument("voteAverage") { }
            )
        ) { backStackEntry ->
            val title = backStackEntry.arguments?.getString("title") ?: ""
            val description = backStackEntry.arguments?.getString("description") ?: ""
            val voteAverage = backStackEntry.arguments?.getString("voteAverage") ?: ""

            MediaDetailScreen(
                name = title,
                description = description,
                imageBackdropUrl = "https://image.tmdb.org/t/p/w500//mSDsSDwaP3E7dEfUPWy4J0djt4O.jpg",
                imagePosterUrl = "https://image.tmdb.org/t/p/w500//39wmItIWsg5sZMyRUHLkWBcuVCM.jpg",
                voteAverage = voteAverage
            ) {
                navController.popBackStack()
            }
        }
    }
}

private fun navigateToMediaDetail(navController: NavHostController, media: Media) {
    navController.navigate("${Screens.DetailMediaScreen}/${media.title ?: media.name ?: ""}/${media.description}/${media.getVoteAverageFormatted()}")
}
```

Reply:
with these changes , i am trying to  provide better handling of navigation arguments and route paths.